demo/src/main/java/com/example/demo/controller/UserController.java

```diff
@@ -0,0 +1,39 @@
+package com.example.demo.controller;
+
+import com.example.demo.model.User;
+import com.example.demo.services.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping(path="api/v1/user")
+public class UserController {
+
+    @Autowired
+    public UserService userService;
+
+    @PostMapping//create
```

Comment:
please remove unnecessary comment // create
it should be obvious from the context